api/hypershift/v1alpha1/hosted_controlplane.go

```diff
@@ -38,6 +40,12 @@ type HostedControlPlaneSpec struct {
 	// If not defined, ReleaseImage is used
 	ControlPlaneReleaseImage *string `json:"controlPlaneReleaseImage,omitempty"`
 
+	// updateService may be used to specify the preferred upstream update service.
+	// By default it will use the appropriate update service for the cluster and region.
```

Comment:
Maybe as a follow-on clean up `s/By default it/By default, the CVO/g`

Reply:
More future alternative wording options:

> By default, lower-level components will use the appropriate update service for the cluster and region.

or:

> When empty, lower-level components will use the appropriate update service for the cluster and region.

to avoid committing to the CVO as the lower-level actor that is filling in an opinion when HostedControlPlane has an empty-string here.